### DISCUSSION TITLE
How to handle one-to-many and many-to-one relationship in OC

### DISCUSSION THREAD

**Discussion by HengzheLi** [OrchardCMS/OrchardCore#7479] [Q&A]
In O1, there was a detail doc https://docs.orchardcore.net/projects/O1/en/latest/Documentation/Creating-1-n-and-n-n-relations/ to tell how to modeling these relationship. 

Is there a doc to tell how to implement them in OC?

**Comment by vitalybrandes** [comment]
Check the Content Picker filed.
